release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -160,9 +160,51 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-machine-config-operator"]
 === Machine Config Operator
 
+[id="ocp-4-13-machine-config-operator-layering"]
+==== Support for adding third party and custom content to {op-system}
+
+You can now use {op-system-first} image layering to add third party agents and other software to cluster nodes. In addition to third party content, you can also use {op-system} image layering to inject configuration. 
+
+For more information, see  xref:../post_installation_configuration/coreos-layering.adoc#coreos-layering[{op-system-first} image layering].
+
+[id="ocp-4-13-machine-config-operator-core"]
+==== Support for setting the `core` user password
+
+You can now create a password for the {op-system} `core` user. In the event that you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC). 
+
+For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#core-user-password_post-install-machine-configuration-tasks[Changing the core user password for node access].
+
 [id="ocp-4-13-nodes"]
 === Nodes
 
+[id="ocp-4-13-nodes-mirror"]
+==== Image registry repository mirroring by tags
+
+You can now pull images from a mirrored registry by using image tags and not only digest specifications. To accomplish this change, the `ImageContentSourcePolicy` (ICSP) object is now deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using digest specifications or an `ImageTagMirrorSet` object (ITMS) to pull images by using image tags.
+
+If you have existing YAML files that you used to create ICSP objects, you can use the `oc adm migrate icsp` command to convert those files to an IDMS YAML file.
+
+For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
+
+For more information on converting existing ICSP objects to IDMS objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
```

Comment:
The link of "For more information on converting existing ICSP objects to IDMS objects, see" not exist. It should be: https://56250--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror-convert_post-install-preparing-for-users